src/app/dashboard/dashboard.component.html

```diff
@@ -78,6 +78,11 @@ <h4 class="card-title">Sault Git Users Stats</h4>
                                   <td><a href="https://www.linkedin.com/in/victorbarretodeoliveira/">victorbarretodeoliveira</a></td>                                 
                                   <td><a href="https://github.com/vbarretoo3">vbarretoo3</a></td>
                                 </tr>
+                                <tr>
+                                    <td>Michael Lajoie  </td>
```

Comment:
nit: Could remove the empty spaces `<td>Michael Lajoie</td>`

Reply:
Done